services/web-client/source/class/qxapp/layout/LayoutManager.js

```diff
@@ -21,15 +21,15 @@ qx.Class.define("qxapp.layout.LayoutManager", {
     this._pane = new qx.ui.splitpane.Pane("horizontal");
 
     const settingsWidth = 500;
-    this._settingsView = new qxapp.components.SettingsView();
+    this._settingsView = new qxapp.components.workbench.SettingsView();
     this._settingsView.set({
       minWidth: settingsWidth*0.5,
       maxWidth: settingsWidth,
       width: settingsWidth*0.75
     });
     this._pane.add(this._settingsView, 0);
 
-    this._workbench = new qxapp.components.Workbench();
+    this._workbench = new qxapp.components.workbench.Workbench();
```

Comment:
Declare _workbench as a member (same way you did with _pan)

Reply:
Done